docs/src/js/routes.tsx

```diff
@@ -22,12 +22,11 @@ import TypographyPage from "./pages/TypographyPage";
 import NotFoundPage from "./pages/NotFoundPage";
 import WelcomePage from "./pages/WelcomePage";
 
-function MainConteinr(props) {
+function MainContainer(props) {
     let {
-        children,
+        children = <WelcomePage />,
```

Comment:
It seems not working as expected.

Default value of children is `null`. So `children` is not set to `<WelcomePage>` since it has value "`null`".